docs-source/spring/content/development/cli/_index.md

```diff
@@ -957,6 +1014,210 @@ GraalVM Compile Commands
             E-mail: obaas_ww@oracle.com
     ```
 
+### User Management
+
+Manage users let you store an unlimited amount of users and apply the access permission using roles into the platform. A user role represents a specific set of tasks or responsibilities assigned to a group of users within the platform. Assigning user roles enables organizations to control the actions each user can perform in the system, streamlining their workflow.
+
+**User Roles**
+
+* ROLE_ADMIN: Users with this role usually have complete access rights, allowing them to manage all platform aspects. Only a user with this role can create new users, search for registered users, change passwords and roles from the other users, and delete users.
+
+* ROLE_CONFIG_EDITOR: This role might be able to edit the Platform configurations. **Currently unused in the environment**.
+
+* ROLE_USER: This role might be able to connect with Admin Service, create and list for applications (namespaces); use bind command; deploy, list and scale workloads (services).
+
+#### Create users
+
+Use the `user create` command to add users to the platform. This command requires the name of the user `username` and the user roles in a comma-separated list.
+
+```cmd
+oractl:>help user create
+NAME
+       user create - Creates a new user in your platform.
+
+SYNOPSIS
+       user create [--username String] --roles String --help
+
+OPTIONS
+       --username String
+       The name you assign to the user during creation. This is the user’s login for the CLI and for the SOC UI, also. The name must be unique across all users in the platform and cannot be changed.
+       [Mandatory]
+
+       --roles String
+       The user's role within the platform. A user must have up to three possible roles provided in a comma-separated list. [ROLE_ADMIN,ROLE_CONFIG_EDITOR,ROLE_USER].
+       [Optional, default = ROLE_USER]
+
+
+...
+
+```
+
+Example:
```

Comment:
"For example" not "example". Even better would be to explain what it is an example of --

"For example, to create a user called xxx with roles xxx:"